apps/docs/components/components/scrollable.md

```diff
@@ -82,9 +82,10 @@ By default `SfScrollable` scroll by one page of items, but can be modified that
 | `buttonsPlacement` | `SfScrollableButtonsPlacement`  | `'block'` | `'block'`, `'floating'`, `'none'`      |
 | `activeIndex` | `number`  |  |       |
 | `reduceMotion` | `boolean`  |  |       |
-| `drag` | `{ sensitivity: number; } | boolean`  |  |       |
+| `drag` | `{ sensitivity: number; containerWidth: boolean; } | boolean`  |  |       |
```

Comment:
No explanation on what `containerWidth` is. Could you provide a brief description? This type could also be a dedicated object like `SfScrollableOnDragEndData`, but we must have an extra sections where we provide detailed description of these objects types.